packages/runtime-handler/src/dev-runtime/route.ts

```diff
@@ -303,25 +303,46 @@ export function functionPathToRoute(
         reply,
         debugMessage,
         debugArgs = [],
+        crossForkLogMessage,
       }: {
         err?: Error | number | string;
         reply?: Reply;
         debugMessage?: string;
         debugArgs?: any[];
+        crossForkLogMessage?: {
+          level: keyof LoggerInstance;
+          args: [string] | [string, number] | [string, string];
+        };
       }) => {
         if (debugMessage) {
           log(debugMessage, ...debugArgs);
           return;
         }
+
+        if (crossForkLogMessage) {
+          if (
+            config.logger &&
+            typeof config.logger[crossForkLogMessage.level] === 'function'
+          ) {
+            config.logger[crossForkLogMessage.level](
+              // @ts-ignore
```

Comment:
I'm not too happy about this but typing here would have been quite the pain and in my opinion would distract from the fact that this is just a pass through.